needy/local_configuration.py

```
@@ -48,6 +50,22 @@ def development_mode(self, library_name):
     def set_development_mode(self, library_name, enable=True):
         self.__set_library_configuration(library_name, 'development_mode', enable)
 
+    def cache(self):
+        if 'cache' in self.__configuration:
+            cache_type = self.__configuration['cache']['type']
+            for candidate in [DirectoryCache]:
+                if cache_type == candidate.type():
+                    return candidate(self.__configuration['cache']['configuration'])
```

I wonder if this should throw or warn for an invalid cache type.